server/controllers/card.js

```diff
@@ -0,0 +1,52 @@
+const Card = require("../models/Card");
+const Column = require("../models/Column");
+const asyncHandler = require("express-async-handler");
+
+// @route POST /card/create
+// @desc Create a card within a column
+// @access Private
+exports.createCard = asyncHandler(async (req, res, next) => {
+  try {
+    const cardToBeCreated = req.body.card;
+    const card = await Card.create(cardToBeCreated);
+    const column = await Column.findById(req.body.columnId).exec();
+    column.cards.push(card);
+    await Column.findByIdAndUpdate(req.body.columnId, column).exec();
+    res.status(201).json(card);
+  } catch (err) {
+    console.error(err);
+    res.status(500).send(err);
+  }
+});
+
+// @route POST /card/updateName
```

Comment:
Instead of having multiple endpoints to update different things, I suggest having one update endpoint (let's say /card/update for example, but technically you can also just create a PUT /card endpoint for best practices) and update validated attributes of any card

Reply:
Good point, I'll apply `/card/update` in this branch for now to make it available to the team today and then I'll use PUT in the upcoming BE: detailed card feature #2  for best practices